sdk/testing/helpers/testing.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2025 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+// TestingT is implemented by *testing.T and potentially other test frameworks.
+type TestingT interface {
```

Comment:
We nw have 3 versions of this (2 same and this). Not sure I understand why?

Reply:
Because this is not an interface anybody consumes as a promise, but it is input, i.e. a requirement. Hence, it is defined where the code lives that consumes it.